components/skus/browser/rs/lib/src/sdk/orders.rs

```diff
@@ -206,8 +222,47 @@ where
         Ok(order)
     }
 
+    #[instrument]
+    // submit_receipt allows for order proof of payment
+    pub async fn submit_receipt(
+        &self,
+        order_id: &str,
+        receipt: &str,
+    ) -> Result<SubmitReceipt, SkusError> {
+        event!(Level::DEBUG, order_id = order_id, "submit_receipt called");
+        let request_with_retries = FutureRetry::new(
+            || async {
+                let mut builder = http::Request::builder();
+                builder.method("POST");
+                builder.uri(format!("{}/v1/orders/{}/submit-receipt", self.base_url, order_id));
```

Comment:
I think we should validate the `order_id` somewhere, otherwise attackers may perform path traversal alike

Reply:
`order_id` is validated on the backend skus APIs.  We require a valid uuid.